packages/c/sshnpd/src/main.c

```diff
@@ -269,19 +278,19 @@ int main(int argc, char **argv) {
   if (res != 0) {
     atlogger_log(LOGGER_TAG, ATLOGGER_LOGGING_LEVEL_ERROR, "Failed to start refresh device entry thread\n");
     exit_res = res;
-    goto cancel_atclient;
+    goto clean_username_keys;
```

Comment:
A bunch of these gotos were just wrong, so I also fixed the memory cleanup control flow.